public/app/features/explore/state/main.ts

```diff
@@ -133,6 +143,24 @@ export function splitClose(itemId: ExploreId): ThunkResult<void> {
   };
 }
 
+export function changeLargerPane(largerExploreId?: ExploreId): ThunkResult<void> {
+  return (dispatch, getState) => {
+    dispatch(splitSizeUpdateAction({ largerExploreId }));
+  };
+}
+
+export function maximizePane(exploreId?: ExploreId): ThunkResult<void> {
+  return (dispatch, getState) => {
+    dispatch(maximizePaneAction({ exploreId }));
+  };
+}
+
+export function evenResizePane(): ThunkResult<void> {
+  return (dispatch, getState) => {
+    dispatch(evenPaneResizeAction());
+  };
+}
+
```

Comment:
given these thunks are not doing anything async and are just proxying call to plain action creators, we should be using actions creators directly instead.